app/src/main/java/com/twilio/video/app/ui/room/RoomActivity.java

```diff
@@ -281,32 +180,48 @@ protected void onCreate(Bundle savedInstanceState) {
         // Setup toolbar
         setSupportActionBar(toolbar);
 
-        // Setup Audio
-        audioManager = (AudioManager) getSystemService(Context.AUDIO_SERVICE);
-        audioManager.setSpeakerphoneOn(true);
-        savedVolumeControlStream = getVolumeControlStream();
-
         // setup participant controller
         participantController = new ParticipantController(thumbnailLinearLayout, primaryVideoView);
         participantController.setListener(participantClickListener());
 
+        displayName = sharedPreferences.getString(Preferences.DISPLAY_NAME, "");
+
         // Setup Activity
         statsScheduler = new StatsScheduler();
-        obtainVideoConstraints();
         requestPermissions();
     }
 
     @Override
     protected void onResume() {
         super.onResume();
-        boolean isAppLinkProvided = checkIntentURI();
-        displayName = sharedPreferences.getString(Preferences.DISPLAY_NAME, null);
-        updateUi(room, isAppLinkProvided);
+        //        boolean isAppLinkProvided = checkIntentURI();
```

Comment:
Let's add a checklist to the PR to the items that need to be re-enabled.

Reply:
Will create follow up tickets that will encompass what is left to be re-enabled.